decentraland/screens/ListLandForSaleScreen.js

```diff
@@ -46,23 +46,21 @@ export class ListLandForSaleScreen extends React.Component {
     const openSellOrders = await getAllOpenSellOrders();
 
     const contracts = getContracts();
-    const { estateContract } = contracts;
+    const { landContract } = contracts;
 
-    const estatesForSale = [];
+    // Showing only parcels for now because of all estates are with blank images
```

Comment:
Why is that?

Reply:
All 10 estates open sell orders from testnet are with IDs that link to a blank image. I've done that as a short term solution.

Reply:
Okay, good to know. (Saw your message on Slack after making my first comment.) Sounds good.